.doc_gen/readmes/language.yml

```diff
@@ -0,0 +1,32 @@
+name: Ruby
+covered_services:
+- cloudtrail
+- cloudwatch
+- codebuild
+- dynamodb
+- ec2
+- elasticbeanstalk
+- elastictranscoder
+- eventbridge
+- iam
+- kms
+- lambda
+- polly
+- sqs
+- sns
+- ses
+- workdocs
+- rds
+- s3
+- secretsmanager
+short_sdk_name: "AWS SDK for Ruby"
+service_developer_guide: https://docs.aws.amazon.com/sdk-for-ruby/v3/developer-guide/welcome.html
+service_api_reference_guide: https://docs.aws.amazon.com/sdk-for-ruby/v3/api/
+language_sdk_reference_guide: https://aws.amazon.com/developer/language/ruby/
+running_code:
+- scenario: "The easiest way to interact with this example code is by invoking a [Scenario](#Scenarios) from your Command Line Interface (CLI). For example, `ruby some_scenario.rb` will invoke `some_scenario.rb`."
```

Comment:
If "your Command Line Interface (CLI)" refers to the AWS CLI and not a generic command line interface, then: 
your Command Line Interface (CLI). --> the AWS Command Line Interface (AWS CLI).

If it is a reference to a generic command line interface, then: 
your Command Line Interface (CLI). --> the command line.